coffeecard/CoffeeCard.Library/Services/TokenService.cs

```diff
@@ -44,28 +44,65 @@ public JwtSecurityToken ReadToken(string token)
             return new JwtSecurityTokenHandler().ReadToken(token) as JwtSecurityToken;
         }
 
+        
         /// <summary>
-        ///     Receives the serialized version of a JWT token as a string. Checks if the JWT token is valid (Based on its lifetime
-        ///     and if it has been used before)
+        /// Receives the serialized version of a JWT token as a string.
+        /// Checks if the JWT token is valid (Based on its lifetime)
         /// </summary>
         /// <param name="tokenString"></param>
-        /// <returns></returns>
-        public async Task<bool> ValidateToken(string tokenString)
+        /// <returns>bool</returns>
+        public async Task<bool> ValidateTokenAsync(string tokenString)
```

Comment:
No asynchronous code in this method. Async can be removed

Reply:
Removed async